oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/MissingLastRevSeekerTest.java

```diff
@@ -248,6 +248,7 @@ public void dispose() {
             dns.merge(b1, EmptyHook.INSTANCE, CommitInfo.EMPTY);
         }
         dns.runBackgroundOperations();
+        Thread.sleep(1001);
```

Comment:
This might still fail when running on nodes with low resources or add unneeded delays when the background operation completes quickly. In oak-search, we make extensive use of the following utility method that takes care of both issues:

https://github.com/apache/jackrabbit-oak/blob/3e315733e3932b77de9606271165190c2b106fa8/oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/TestUtil.java#L247

Reply:
Adding code with assertEventually(..) still fails on local, even if I add 100s as timeout. With no timeout it still fails.